src/main/java/tk/shanebee/hg/game/GamePlayerData.java

```diff
@@ -305,6 +309,24 @@ public void join(Player player, boolean command) {
                 freeze(player);
                 kills.put(player, 0);
 
+                if (Config.enableleaveitem){
+                    ItemStack bed = new ItemStack(Material.getMaterial(Config.leaveitemtype), 1);
+                    ItemMeta commeta = bed.getItemMeta();
+                    commeta.setDisplayName(lang.leave_game);
+                    bed.setItemMeta(commeta);
+                    player.getInventory().setItem(8, bed);
+                }
+
+                if (Config.enableforcestartitem){
+                    if (player.hasPermission("hg.forcestart")){
+                        ItemStack start = new ItemStack(Material.getMaterial(Config.forcestartitem), 1);
+                        ItemMeta meta = start.getItemMeta();
+                        meta.setDisplayName(lang.force_start);
+                        start.setItemMeta(meta);
+                        player.getInventory().setItem(0, start);
+                    }
+                }
```

Comment:
These two conditions could probably be done on the same line, also watch your spacing here
```suggestion
                if (Config.enableforcestartitem && player.hasPermission("hg.forcestart") {
                    ItemStack start = new ItemStack(Material.getMaterial(Config.forcestartitem), 1);
                    ItemMeta meta = start.getItemMeta();
                    meta.setDisplayName(lang.force_start);
                    start.setItemMeta(meta);
                    player.getInventory().setItem(0, start);
                }
```